src/main/scala/cromwell/binding/values/WdlMap.scala

```diff
@@ -80,11 +80,11 @@ case class WdlMap(wdlType: WdlMapType, value: Map[WdlValue, WdlValue]) extends W
     collected.flatten.toSeq
   }
 
-  override def getHash(implicit hasher: FileHasher): Hash = {
+  override def getHash(implicit hasher: FileHasher): SymbolHash = {
     val hashedMap = value map {
       case (k, v) => k.getHash -> v.getHash
     }
     val concatenatedMap = TreeMap(hashedMap.toArray: _*).foldLeft("") { (acc, kv) => acc + kv._1 + kv._2 }
-    (getClass.getCanonicalName+concatenatedMap).md5Sum
+    SymbolHash((getClass.getCanonicalName+concatenatedMap).md5Sum)
```

Comment:
the spacing around the `+` is consistently missing in the `SymbolHash.apply`s here and below